test/infrastructure/docker/internal/controllers/backends/docker/dockermachine_backend.go

```diff
@@ -254,12 +275,24 @@ func (r *MachineBackendReconciler) ReconcileNormal(ctx context.Context, cluster
 			// Run the bootstrap script. Simulates cloud-init/Ignition.
 			if err := externalMachine.ExecBootstrap(timeoutCtx, bootstrapData, format, version, dockerMachine.Spec.Backend.Docker.CustomImage); err != nil {
 				conditions.MarkFalse(dockerMachine, infrav1.BootstrapExecSucceededCondition, infrav1.BootstrapFailedReason, clusterv1.ConditionSeverityWarning, "Repeating bootstrap")
+				v1beta2conditions.Set(dockerMachine, metav1.Condition{
+					Type:    infrav1.DevMachineDockerBootstrapExecSucceededV1Beta2Condition,
+					Status:  metav1.ConditionFalse,
+					Reason:  infrav1.DevMachineDockerBootstrapExecNotSucceededV1Beta2Reason,
+					Message: "Failed to exec DockerMachine bootstrap",
```

Comment:
Do we want to include the error here and below? (like we did in some other cases in this PR)

Reply:
Yeah was just wondering. I think in the other cases it wasn't mirrored. But I don't mind. Resolving